resources/articles/migration-service.md

```diff
@@ -27,6 +27,8 @@ response.
 
 ![Migration service](/images/migration-service.png)
 
+A sample migration service implementation can be found on GitHub: https://github.schibsted.io/spt-identity/fake-migration-service
```

Comment:
This should be in the public repo. The migration service is not for only Schibsted companies.